docs/pip/compatibility.md

```diff
@@ -183,7 +183,7 @@ In other words, uv inverts the default, requiring explicit opt-in to installing
 Python, which can lead to breakages and other complications, and should only be done in limited
 circumstances.
 
-For more, see ["Installing into arbitrary Python environments"](./README.md#installing-into-arbitrary-python-environments).
+For more, see ["Using arbitrary Python environments"](./environments.md#using-arbitrary-python-environments).
```

Comment:
I think? Will be annoying to keep this in-sync with `PIP_COMPATIBILITY.md`, though maybe the goal is to remove that.

Reply:
Yes we'll move `PIP_COMPATIBILITY.md` to here and leave a placeholder with a link,